openedx/features/course_bookmarks/plugins.py

```diff
@@ -14,9 +15,9 @@ class CourseBookmarksTool(CourseTool):
     @classmethod
     def is_enabled(cls, request, course_key):
         """
-        Always show the bookmarks tool.
+        The bookmarks tool is only enabled for enrolled users.
```

Comment:
Thinking this through more, I don't see why staff should have access to the bookmarks tool if they aren't enrolled. I've changed it to just check the regular ``is_enrolled`` method.